daml-assistant/integration-tests/src/Main.hs

```diff
@@ -59,15 +59,10 @@ quickstartTests :: FilePath -> FilePath -> TestTree
 quickstartTests quickstartDir mvnDir = testGroup "quickstart"
     [ testCase "daml new" $
           callProcessQuiet "daml" ["new", quickstartDir]
-    , testCase "daml init" $ withCurrentDirectory quickstartDir $
-          callProcessQuiet "daml" ["init"]
-    , testCase "daml package" $ withCurrentDirectory quickstartDir $
-          -- This location is assumed by the codegen in the quickstart example.
-          callProcessQuiet "daml" ["package", "-o", "target/daml/iou.dar"]
     , testCase "daml build " $ withCurrentDirectory quickstartDir $
           callProcessQuiet "daml" ["build"]
-    , testCase "daml test" $ withCurrentDirectory quickstartDir $
-          callProcessQuiet "daml" ["test", "daml/Main.daml"]
+    , testCase "daml damlc test" $ withCurrentDirectory quickstartDir $
```

Comment:
I opened https://github.com/digital-asset/daml/issues/743 to track the changes to `damlc test` before we add `daml test` back.

Reply:
Thanks!